### DISCUSSION TITLE
Multi-Select for Deploy Token Scope

### DISCUSSION THREAD

**Discussion by ajoberstar** [ato/clojars-web#771] [Ideas]
As I'm starting to update my projects to use deploy token's I've run into a use-case for multi-selecting the deploy token scope. It's good that we have the `*` options, but `clj-new` kind of forces you to use a unique group for each template.

For example, I'd like to make a token with access to:
- `dev.clojurephant/*`
- `clojurephant-clj-app/*`
- `clojurephant-clj-lib/*`
- `clojurephant-cljs-app/*`

But not have it access the other scopes.

If this is possible and I'm just mistaken, would love to be corrected! Appreciate all you do to make Clojars available for the community.

**Comment by tobias** [comment]
Hi @ajoberstar! 

You are correct - it currently isn't possible to have a token that can deploy to different groups without having a scope of `*`. I wouldn't be opposed to someone adding support for this, but it would be a bit of work since it would require some changes to the data model.
